dango/dex/src/cron.rs

```diff
@@ -384,6 +384,17 @@ fn clear_orders_of_pair(
 
     #[cfg(feature = "metrics")]
     {
+        let num_trades = bids.len() + asks.len();
```

Comment:
Previously we use `filling_outcomes.len()`. It should equal the lengths of `bids` and `asks` combined.